requirements/requirements.txt

```diff
@@ -1,3 +1,12 @@
+# version fix
+PyYAML~=5.4
+python-dateutil==2.6.0
+matplotlib~=3.3.4
+numpy==1.19.5
+# dependencies
+TTS ~= 0.6.1
+# neon
 neon-utils~=0.5
 ovos-plugin-manager~=0.0.2
-# TODO: Add any dependencies
\ No newline at end of file
+# huggingface
+huggingface-hub
```

Comment:
Is there a loose version spec for this dependency?

Reply:
Agreed, No. It is latest version, but it has common dependencies that may brake later. I will change it